tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -50,7 +52,22 @@ class GTMaskLoader(Enum):
 
 
 class SegmentationRepresentation(BaseRepresentation):
-    pass
+    def to_polygon(self, segmentation_colors=None):
+        polygons = defaultdict(list)
+        mask = self._encode_mask(self.mask, segmentation_colors) if segmentation_colors else self.mask
+        if len(mask.shape) == 3:
```

Comment:
what if mask already argmaxed and have shape [h, w, 1]?

Reply:
add condition